detect-nuget-inspector/detect-nuget-inspector/Inspection/Util/ProjectFile.cs

```diff
@@ -28,6 +28,12 @@ public static ProjectFile Parse(string projectLine)
             {
                 file.TypeGUID = MiddleOfString(leftSide, "Project(\"".Length, "\")".Length);
             }
+            else
```

Comment:
This is a very weird situation which required a fix when the there are multiple props file which are mentioned under sln file ProjectSection, as an example
ProjectSection(SolutionItems) = preProject ..\samples\Directory.Build.props = ..\samples\Directory.Build.props ..\samples\Directory.Packages.props = ..\samples\Directory.Packages.props EndProjectSection.
This is a different props file which was present under samples folder which is not in the root directory for sln file, so this logic would go up each directory to check if props file exists in that directory and parse it if it does.